Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -950,433 +906,440 @@
+    <RegEx find="([sS])ažalj?eva(m|š|mo|te|ju|ti|li|le|la|lo|jući|juća|juće|nje)" replaceWith="$1ažalijeva$2" />
+    <RegEx find="\b([sS])edi(m|š|i|imo|ite|e|eći|ili|ile|ilo|iše)\b" replaceWith="$1jedi$2" />
+    <RegEx find="\b([sS])istem(a|e|i|u|om|ima)?\b" replaceWith="$1ustav$2" />
+    <RegEx find="([sS])mećar(a|e|i|ski|ska|sko|skom|skoj|skih)?" replaceWith="$1metlar$2" />
+    <RegEx find="\b([sS])mj?e(m|š|mo|te|šna|šne|šni|šno|šnom|šnoj|šnu|im|ima)\b" replaceWith="$1mije$2" />
+    <RegEx find="\b([sS])mej(u|e|em|eš|emo|ete|anje|anju|ati|ali|ale|ao|ala|alo)" replaceWith="$1mij$2" />
+    <RegEx find="([sS])mest(i|ili|ile|io|ila|ilo|iti|ivši)" replaceWith="$1mjest$2" />
+    <RegEx find="([sS])pasava(m|š|mo|te|ju|li|nje|nju|nja)?" replaceWith="$1pašava$2" />
+    <RegEx find="([sS])pri?j?ečava(m|š|mo|te|ju|li|še|nje)?" replaceWith="$1prječava$2" />
+    <RegEx find="([sS])prj?eči(m|š|mo|te|li|še|o|la|li|le|lo|ti)?" replaceWith="$1priječi$2" />
+    <RegEx find="([sS])prj?eče" replaceWith="$1priječe" />
+    <RegEx find="([sS])umlja(m|š|še|mo|te|ti|ju|jući|li|le|lo)?" replaceWith="$1umnja$2" />
+    <RegEx find="\b([sS])eti(t|o|m|š|la|li|le|ti)?\b" replaceWith="$1jeti$2" />
+    <RegEx find="([sS])ever(a|u|om|ni|nom|ac)?" replaceWith="$1jever$2" />
+    <RegEx find="([sS])eća(m|š|mo|te|ju|li|nje)?" replaceWith="$1jeća$2" />
+    <RegEx find="([sS])hvata(m|š|mo|te|ju|jući|la|li|le|lo|ti|nje)?" replaceWith="$1hvaća$2" />
+    <RegEx find="([sS])led(eće|ećeg|eća|eću|ećom|ećoj|ećem|eći|ećom|ećima|ećim|ećih)?" replaceWith="$1ljed$2" />
+    <RegEx find="\b([sS])ljed(a|e|i|u|om|im|iš|imo|ite|e|ili)?\b" replaceWith="$1lijed$2" />
+    <RegEx find="([rR])edosljed(a|e|u|om)?" replaceWith="$1edoslijed$2" />
+    <RegEx find="([sS])meh(a|u|om)?" replaceWith="$1mijeh$2" />
+    <RegEx find="sopstven(a|e|i|o|u|om|oj|im|og|ima|ost|ošću)?" replaceWith="vlastit$1" />
+    <RegEx find="Sopstven(a|e|i|o|u|om|oj|im|og|ima|ost|ošću)?" replaceWith="Vlastit$1" />
+    <RegEx find="spakuje(m|eš|mo|te)?" replaceWith="spakira$1" />
+    <RegEx find="([sS])reč(a|e|i|u|om|ama)" replaceWith="$1reć$2" />
+    <RegEx find="([sS])re[čć](an|na|ni|nu|nik|nika|nikom|nica|nicu|nicom|no|nom|noj|niji|nijeg|nijem|nija|niju|nijom|nijoj)" replaceWith="$1ret$2" />
+    <RegEx find="stomak(u|om)?" replaceWith="trbuh$1" />
+    <RegEx find="stomač(na|ne|ni|no|nu|noj|nom|nima)" replaceWith="trbuš$1" />
+    <RegEx find="\b([sS])trj?el(a|e|i|u|o|om|ac|cu|ci|cima|ama)" replaceWith="$1trijel$2" />
+    <RegEx find="sujeverj(a|e|u|ima|em)" replaceWith="praznovjerj$1" />
+    <RegEx find="([sS])used(a|e|i|o|u|ama|ima|ski|skim|ima)?" replaceWith="$1usjed$2" />
+    <RegEx find="suštin(e|i|om)" replaceWith="biti" />
+    <RegEx find="([sS])vat(am|aš|a|amo|ate|aju|ajući|ali|ale|alo|aše)" replaceWith="$1hvać$2" />
+    <RegEx find="([sS])vedo(čim|čiš|či|čimo|čite|če|k|ke|ka|ku|kom|ci|cima|kinja|kinju|kinji|čanstvo|čanstva|čili|čio|čila|čiti)" replaceWith="$1vjedo$2" />
```

Comment:
If  you do not need them in  the "replaceWith" string, you do not need a group. For example:
```
<RegEx find="([uU])propaš[čć]ava" replaceWith="$1propaštava" />
```
You  need the "u" or  "U", because you want to keep the original casing.
But  you don't  care about "č" or "ć".